.gitignore

```diff
@@ -44,6 +44,7 @@ sdks/python/apache_beam/portability/common_urns.py
 **/*.iml
 **/*.ipr
 **/*.iws
+**/out/**/*
```

Comment:
@kennknowles I noticed that you closed #4626 without merging, is there something better that we should be doing here instead?

Reply:
@lukecwik I just removed the commit that did the out/ part until we decide what to do with this. I will merge the other one since I plan this to be rebased during the review of the new sketch.